server/api/version_test.go

```diff
@@ -19,67 +19,66 @@ import (
 	"io"
 	"os"
 	"path/filepath"
-	"strings"
+	"testing"
 	"time"
 
-	. "github.com/pingcap/check"
 	"github.com/pingcap/failpoint"
+	"github.com/stretchr/testify/require"
 	"github.com/tikv/pd/pkg/assertutil"
 	"github.com/tikv/pd/pkg/testutil"
 	"github.com/tikv/pd/server"
 	"github.com/tikv/pd/server/config"
 )
 
-var _ = Suite(&testVersionSuite{})
-
-func checkerWithNilAssert(c *C) *assertutil.Checker {
-	checker := assertutil.NewChecker(c.FailNow)
+func checkerWithNilAssert(re *require.Assertions) *assertutil.Checker {
+	checker := assertutil.NewChecker(func() {
+		re.FailNow("should be nil")
+	})
 	checker.IsNil = func(obtained interface{}) {
-		c.Assert(obtained, IsNil)
+		re.Nil(obtained)
 	}
 	return checker
 }
 
-type testVersionSuite struct{}
-
-func (s *testVersionSuite) TestGetVersion(c *C) {
+func TestGetVersion(t *testing.T) {
 	// TODO: enable it.
-	c.Skip("Temporary disable. See issue: https://github.com/tikv/pd/issues/1893")
+	t.Skip("Temporary disable. See issue: https://github.com/tikv/pd/issues/1893")
```

Comment:
Should we remove it or fix it? This test is disabled for a long time.

Reply:
I think we can remove or fix it in another PR.